solr/modules/opentelemetry/src/java/org/apache/solr/opentelemetry/OtelTracerConfigurator.java

```diff
@@ -49,14 +48,13 @@ public OtelTracerConfigurator() {
   }
 
   @Override
-  public Tracer getTracer() {
-    return TraceUtils.getGlobalTracer();
+  public void init(NamedList<?> args) {
+    prepareConfiguration(args);
   }
 
   @Override
-  public void init(NamedList<?> args) {
-    prepareConfiguration(args);
-    AutoConfiguredOpenTelemetrySdk.initialize();
+  public OpenTelemetry createOtel() {
+    return AutoConfiguredOpenTelemetrySdk.builder().build().getOpenTelemetrySdk();
```

Comment:
notice the change to *not* call initialize.  That method arranges for GlobalOpenTelemetry to be set.  I only want that set in TracerConfigurator.